packages/documentation/src/stories/getting-started/compatibility/compatibility.docs.mdx

```diff
@@ -0,0 +1,95 @@
+import { Meta } from '@storybook/blocks';
+import * as GettingStartedStories from './compatibility.stories';
+
+<Meta of={GettingStartedStories} />
+
+# Compatibility
+
+<p id="compatibility-desc">List of compatible version numbers with external dependencies.</p>
+
+<table aria-describedby="compatibility-desc" className="table caption-top table-responsive">
```

Comment:
The class `table-responsive` should be placed on a wrapping `div` around the table. Otherwise it will not work as intended: https://getbootstrap.com/docs/5.3/content/tables/#responsive-tables.

Reply:
Ah true, I was confused by the name of this class. It makes sense. Sorry I didn't read well your comment.